internal/policies/manager.go

```diff
@@ -23,6 +23,32 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+/*
+
+ADSys is expected to apply configuration policies, not enforce them. We are responsible for
```

Comment:
I would put that on top, so that they show up as package documentation (https://go.dev/doc/comment).